lib/src/rules/prefer_is_empty.dart

```diff
@@ -133,7 +133,7 @@ class _Visitor extends SimpleAstVisitor<void> {
 
     // Don't lint if we're in a const constructor initializer.
     final constructorInitializer =
-        search.parent!.thisOrAncestorOfType<ConstructorInitializer>();
+        search.parent?.thisOrAncestorOfType<ConstructorInitializer>();
```

Comment:
Another place where `parent?.` can safely be removed.

Reply:
👍